bundles/binding/org.openhab.binding.zibase/src/main/java/org/openhab/binding/zibase/internal/ZibaseBindingConfigReceiver.java

```diff
@@ -42,13 +42,46 @@
             ZbProtocol.X2D868.toString(), ZbProtocol.X2D868ALRM.toString(), ZbProtocol.X2D868BOAC.toString(),
             ZbProtocol.X2D868INSH.toString(), ZbProtocol.X2D868PIWI.toString(), ZbProtocol.ZWAVE.toString(), };
 
+    /**
+     * X10 formed id (Z prefix removed for ZWave items
+     */
+    protected String x10Id;
+
+    /**
+     * Zwave indicator used for some Zibase command (eg. getState() )
+     */
+    protected Boolean isZWave = false;
+
     /**
      * Constructor
-     * 
+     *
      * @param configParameters
      */
     public ZibaseBindingConfigReceiver(String[] configParameters) {
         super(configParameters);
+
+        // handle Zwave case : items' id begin with Z, but some zibase call must be called
+        // with the X10 version of the ID (no Z prefix) and a special flag
+        logger.debug("Item protocol : {}", this.getProtocol());
+
+        if (this.getProtocol().equals(ZbProtocol.ZWAVE.toString())) {
```

Comment:
You should consider dropping the occurrences of `this` where they are meaningless (as they are here).